src/controllers/movies.js

```diff
@@ -0,0 +1,164 @@
+const { Prisma } = require("@prisma/client");
+const prisma = require("../utils/prisma");
+
+const getAllMovies = async (req, res) => {
+  console.log("get movies");
+
+  if (!req.query.runtimeLt && !req.query.runtimeGt) {
+    const movies = await prisma.movie.findMany({
+      include: {
+        screenings: true,
+      },
+    });
+
+    res.status(200).json({
+      movies,
+    });
+  }
+
+  if (req.query.runtimeLt) {
+    const movies = await prisma.movie.findMany({
+      where: {
+        runtimeMins: {
+          lt: Number(req.query.runtimeLt),
+        },
+      },
+    });
+
+    res.status(200).json({
+      movies,
+    });
+  }
+
+  if (req.query.runtimeGt) {
+    const movies = await prisma.movie.findMany({
+      where: {
+        runtimeMins: {
+          gt: Number(req.query.runtimeGt),
+        },
+      },
+    });
+
+    res.status(200).json({
+      movies,
+    });
+  }
+
+  if (req.query.runtimeLt && req.query.runtimeGt) {
+    // + try
+    const movies = await prisma.movie.findMany({
+      where: {
+        AND: [
+          {
+            runtimeMins: {
+              lt: req.query.runtimeLt,
+            },
+          },
+          {
+            runtimeMins: {
+              gt: req.query.runtimeGt,
+            },
+          },
+        ],
+      },
+    });
+
+    res.status(200).json({
+      movies,
+    });
+  }
+};
```

Comment:
Nice work on the logic. The next step here would be to make sure you were only writing `prisma.movie.findMany` one time, which would mean trying to construct the query object first